docs/concept-exercises.md

```diff
@@ -51,7 +51,9 @@ An exercise has the following files. In the browser they will show at the releva
 
 ### `.docs/introduction.md`
 
-This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise, and provide a short, concise introduction to the concept(s). The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms that the student can Google if they so want, is preferable over including any code samples or an extensive description. For example we might describe a string as a "Sequence of Unicode characters" or a "series of bytes" or "an object". Unless the student needs to understand the details of what those mean to be able to solve the exercise we should not give more info in this introduction - instead allowing the student to Google, ignore, or map their existing knowledge.
+This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise, and provide a short, concise introduction to the concept(s). The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms, that the student can Google if they so want, is preferable over an extensive description. The exception to this is syntax, which we don't want the student to need Google, so code samples should be included whenever syntax is introduced.
+
+As an example, the introduction for a "strings" exercise might describe a string as a "Sequence of Unicode characters" or a "series of bytes" or "an object". Unless the student needs to understand the details of what those mean to be able to solve the exercise, we should not give more info in this introduction - instead allowing the student to Google, ignore, or map their existing knowledge. However, as the exercise introduces new syntax, a code sample should be included to show the syntax for defining a string.
```

Comment:
Consider: "the introduction **to** a "strings" exercise".  (not "for"). 

Consider rewording the sentences that begin with "Unless the student needs " to this:
"If the student can solve the exercise with a brief description of strings, then we do not need to provide more information. A code sample should be included to show the syntax for defining a string when the exercise introduces the new syntax."

(I'm unclear about your intent here... are you implying that the quoted introductions are too verbose?  Or just right?)

Reply:
The intent is that we show that the introduction text should not be going into great detail on the technical implementation. Does that make sense?

Reply:
That makes sense; it's not what the current text communicated IMO, however.